jsapp/js/components/formLanding.js

```diff
@@ -46,7 +46,6 @@ class FormLanding extends React.Component {
       nextPagesVersions: [],
       anonymousSubmissions: false,
       anonymousPermissions: [],
-      shouldShowNewFeatureDialog: false,
```

Comment:
Just wanted to highlight this to make sure you're removing it on purpose. I assume so, but wanted to check.

Reply:
Yeah, I removed it because there is no place that used it